src/store/plugins/load-layersconfig-on-lang-change.js

```diff
@@ -28,25 +30,38 @@ function loadLayersConfig(lang) {
     })
 }
 
-const loadLayersAndTopicsConfigAndDispatchToStore = (store) => {
-    loadLayersConfig(store.state.i18n.lang)
-        .then((layersConfig) => {
-            store.dispatch('setLayerConfig', layersConfig)
-            loadTopicsFromBackend(layersConfig).then((topicsConfig) => {
-                store.dispatch('setTopics', topicsConfig)
-                if (store.state.topics.current) {
-                    loadTopicTreeForTopic(store.state.i18n.lang, store.state.topics.current).then(
-                        (tree) => store.dispatch('setTopicTree', tree)
-                    )
-                } else {
-                    store.dispatch(
-                        'changeTopic',
-                        topicsConfig.find((topic) => topic.id === 'ech')
-                    )
-                }
-            })
-        })
-        .catch((error) => log.error(error))
+const loadLayersAndTopicsConfigAndDispatchToStore = async (store) => {
+    try {
+        // adding vector tile backend through a hardcoded entry (for now)
+        // this should be removed as soon as the backend delivers a proper configuration
+        // for our vector tile background layer
+        const bgVectorLayer = new VectorLayer(VECTOR_TILES_STYLE_ID, 1.0, VECTOR_TILES_STYLE_URL)
```

Comment:
See comment above as this is here already hardcoded, I would put the attributionName and co here and not in the component.
From where comes from the vector tiles backend ?

Reply:
So now all specific code is on this line (hardcoded values and such).
I don't really know how the backend is served, I'm just aware of this style ID and that if I put that in a Maplibre instance, cool maps are rendered 😉